<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face recognizer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/FaceRecognizer</normaloff>:/image/FaceRecognizer</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Operate</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="separator"/>
    <addaction name="actionCamera"/>
    <addaction name="actionRegister"/>
    <addaction name="actionRecognizer"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSet_model_path"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout_A"/>
    <addaction name="actionUpdate_U"/>
   </widget>
   <widget class="QMenu" name="menuSource">
    <property name="title">
     <string>Source</string>
    </property>
    <addaction name="actionCarmera"/>
    <addaction name="actionFile"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuSource"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
  </widget>
  <action name="actionStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="statusTip">
    <string>Start</string>
   </property>
  </action>
  <action name="actionDetector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Detector</string>
   </property>
   <property name="statusTip">
    <string>Detector</string>
   </property>
  </action>
  <action name="actionLandmarker">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Landmarker</string>
   </property>
   <property name="statusTip">
    <string>Landmarker</string>
   </property>
  </action>
  <action name="actionRecognizer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Recognizer</string>
   </property>
   <property name="statusTip">
    <string>Recognizer</string>
   </property>
  </action>
  <action name="actionSet_model_path">
   <property name="text">
    <string>Set model path</string>
   </property>
   <property name="statusTip">
    <string>Set model path</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
   <property name="statusTip">
    <string>Register</string>
   </property>
  </action>
  <action name="actionAbout_A">
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpdate_U">
   <property name="text">
    <string>Update(&amp;U)</string>
   </property>
   <property name="iconText">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
   <property name="statusTip">
    <string>Update</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Camera</string>
   </property>
   <property name="iconText">
    <string>Display Camera</string>
   </property>
   <property name="toolTip">
    <string>Display Camera</string>
   </property>
   <property name="statusTip">
    <string>Display Camera</string>
   </property>
  </action>
  <action name="actionCarmera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Carmera</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
